mitosheet/mitosheet/step_performers/export_to_file.py

```diff
@@ -17,6 +17,10 @@
 from mitosheet.step_performers.step_performer import StepPerformer
 from mitosheet.step_performers.utils import get_param
 
+from mitosheet.utils import write_to_excel
```

Comment:
Potentially, because we're using this as part of the step performer, I'd want to version it with the rest of the public interface. 

But in this case, we aren't actually exposing this function to users. Also, there are less strong backwards compatibility constraints b/c it won't cause errors in the rest of the analysis to e.g. change the formatting. So I'm cool with this as is!

Reply:
Sounds good!